apps/heureka/src/components/Services/useFetchServiceImageVersionIssues.ts

```diff
@@ -45,6 +47,7 @@ export const useFetchServiceImageVersionIssues = ({
           issueMatchFilter: {
             serviceCcrn: [serviceCcrn],
           },
+          issuesFilter: searchTerm ? { search: [searchTerm] } : undefined,
           orderByIssueSeverity: [
```

Comment:
Static attributes (orderByIssueSeverity, orderBySeverity, orderByTrd, fetchPolicy) should be set as default to the useGetServiceImageVersionIssuesLazyQuery. See example here:
https://github.com/cloudoperators/juno/blob/893f3fd537ea7e0c9d2d846cea2645a6c99708b1/apps/heureka/src/components/Services/useFetchServices.ts#L21